openshift-ci/build-root/Dockerfile.travislogin

```diff
@@ -0,0 +1,23 @@
+# Dockerfile to bootstrap build and test in openshift-ci
+
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13
```

Comment:
Instead of fetching image stream again, I would say let's implement it as https://github.com/openshift/odo/blob/master/openshift-ci/build-root/source-image/Dockerfile#L1

Reply:
We can not create a separate test container with out pulling the image. BTW the travis test container has certain steps to follow and i do not pollute the regular test container with the travis steps.

Ofcourse as per the requirement i may need https://github.com/openshift/odo/blob/master/openshift-ci/build-root/source-image/Dockerfile#L1 in my corresponding release job pr for copying the oc binary.

Reply:
> BTW the travis test container has certain steps to follow and i do not pollute the regular test container with the travis steps.

make sense 👍 

But I would say once you get the overall workflow to do this, please ensure if we can have only one dockerfile for build. Other than that I am good with the changes.

/lgtm

Reply:
Corresponding ci pr created - https://github.com/openshift/release/pull/10942

Reply:
> But I would say once you get the overall workflow to do this, please ensure if we can have only one dockerfile for build.

@prietyc123 AFAIU It is not even possible because if i am installing packages in the source image for example https://github.com/openshift/odo/blob/master/openshift-ci/build-root/source-image/Dockerfile#L1 then it won't be propagated to the main build container. As per my requirement i do not need binary copy paste through the docker file, i need to install the packages for travis login in the main test container.